.github/workflows/ci.yml

```diff
@@ -134,6 +158,12 @@ jobs:
       - name: Setup .NET
         uses: ./.github/actions/setup-dotnet
 
+      - uses: actions/setup-dotnet@v4
```

Comment:
We already have setup-botnet above, we should integrate theses settings into it

Reply:
Not sure we only need this for .NET 10 tests, the other we run in a few more places.

What do you suggest add an extra parameter to setup-dotnet?